lib/generators/scenic/view/templates/db/migrate/create_view.erb

```diff
@@ -1,5 +1,5 @@
 class <%= migration_class_name %> < ActiveRecord::Migration
   def change
-    create_view :<%= plural_name %><%= ", materialized: true" if materialized? %>
+    create_view "<%= plural_name %>"<%= ", materialized: true" if materialized? %>
```

Comment:
It would be nice if we stuck with symbols unless we need the strings. I don't know how much I care, but it would be nice.